src/utils/supabase.ts

```diff
@@ -841,3 +845,98 @@ export const getChannel = async (
     message: "Successfully fetched the channel!",
   };
 };
+
+export const getTutors = async (
+  query: TutorQuery
+): Promise<SupabaseResponse<TutorSelect[]>> => {
+  const tutorsResponse = await supabase
+  .from("tutors")
+  .select("*")
+  .eq("contact_id", query.contact_id);
```

Comment:
We'll need to extract this line and utilize the `addQueryFilters` function above, similar to how the `getContacts` method works. This is because the `TutorQuery` type contains several optional values and we want to query the db based on only/all the values given. For example, since `contact_id` is optional, we can't add the filter statically like this as it could be undefined and lead to problems later.

The `addQueryFilters` should also be updated to separate all uuid's from normal query types. I'll look into this later and we can further discuss it during our meeting today.